File: src/main/java/no/nav/foreldrepenger/boot/conditionals/OnDevCondition.java
package no.nav.foreldrepenger.boot.conditionals;

import static no.nav.foreldrepenger.boot.conditionals.Cluster.devClusters;

import org.springframework.core.type.AnnotatedTypeMetadata;

public class OnDevCondition extends OnClusterCondition {

    @Override
    protected Cluster[] clusters(AnnotatedTypeMetadata md) {
        return devClusters();
    }
}

File: src/main/kotlin/no/nav/boot/conditionals/OnDevCondition.kt
package no.nav.boot.conditionals

import no.nav.boot.conditionals.Cluster.Companion.devClusters
import org.springframework.core.type.AnnotatedTypeMetadata

class OnDevCondition : OnClusterCondition() {
    override fun clusters(md: AnnotatedTypeMetadata) = devClusters()
}